velox/substrait/SubstraitToVeloxExpr.h

```diff
@@ -66,4 +71,6 @@ class SubstraitVeloxExprConverter {
   std::unordered_map<uint64_t, std::string> functionMap_;
 };
 
+enum VeloxFunName { RowConstructor };
```

Comment:
> Are you planning to hard-code all function in this enum? Why is this needed? Why can't we simply query the function registry? Would you explain the overall problem you are trying to solve, so we can find a better solution?

@mbasmanova  Actually, I use "if (veloxFunction == "row_constructor")" directly, And you said this way is hard-code, So I just use enum to avoid hard-code.  Could you give me some suggestions? Because I saw some codes in velox are also use this style like "if (expr->getFunctionName() == "row_constructor")".

Here is former conversation.
![image](https://user-images.githubusercontent.com/14065538/179714969-b0a69a27-5230-421b-8eb4-8137b650d355.png)